types/react-router-redux/index.d.ts

```diff
@@ -25,15 +25,18 @@ export interface ConnectedRouterProps<State> {
 }
 export class ConnectedRouter<State> extends React.Component<ConnectedRouterProps<State>> {}
 
-export const LOCATION_CHANGE: string;
```

Comment:
I think this change has already been accomplished by https://github.com/DefinitelyTyped/DefinitelyTyped/pull/18566.